src/bitcoin/Transaction.mo

```diff
@@ -0,0 +1,187 @@
+import Array "mo:base/Array";
+import Iter "mo:base/Iter";
+import Nat32 "mo:base/Nat32";
+import Blob "mo:base/Blob";
+import Hash "../Hash";
+import Script "./Script";
+import Common "../Common";
+import Types "./Types";
+
+module {
+  // Representation of a TxIn of a Bitcoin transaction. A TxIn is linked to a
+  // previous transaction output given by prevOutput.
+  public class TxIn(prevOutput : Types.OutPoint, sequence : Nat32) {
```

Comment:
I would rename it to `TxInput`. It makes it clearer, in my opinion, and we can afford the few additional characters.   :-)